.github/workflows/triage.yml

```diff
@@ -0,0 +1,25 @@
+name: Triage issues
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+  pull_request:
+jobs:
+  label_issues:
+    if: github.actor!= 'dependabot[bot]'
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Label issues
+        uses: andymckay/labeler@e6c4322d0397f3240f0e7e30a33b5c5df2d39e90
+        with:
+          add-labels: "triage"
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think we need this. It should be added by our issue templates, which I think is a bit cleaner